### PR TITLE
Add menu component (without routing);

### PR DESCRIPTION
**Pull request by skrzepij**
Additionally:
- Add different bg video quality depending on device size;
- Remove unused Header component;

### CODE REVIEW

File: src/components/homeHero/homeHero.module.scss [hunk 1]
```diff
@@ -11,17 +11,28 @@
...
-  background-color: black;
-  object-fit: cover;
+  &__video, &__background {
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100vw;
+    min-height: 100vh;
+    background-color: black;
```
Comment (MateuszWasik) [inline src/components/homeHero/homeHero.module.scss]: $uc-black ? :)

File: src/components/menu/menu.module.scss [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  border: 1px solid transparentize($uc-white, .9);
+  border-bottom: none;
+  border-top-left-radius: 300px;
+  border-top-right-radius: 300px;
+  z-index: 2;
+  overflow: hidden;
+  transform: translateY(0) scale(1);
+  transition: transform .3s ease-in-out;
+
+  &.collapsed {;
```
Comment (MateuszWasik) [inline src/components/menu/menu.module.scss]: `;` :)

File: src/components/menu/menu.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      path: '#3',
+    },
+  ]
+
+  const generateItems = (items: MenuItem[]): ReactNode => {
+    return items.map(item => {
+      return (
+        <li className={styles.menu__item} key={item.path}>
+          <a href={item.path} className={styles.menu__link}>
+            <span className={styles.menu__link__icon}>{ item.icon }</span>
```
Comment (MateuszWasik) [inline src/components/menu/menu.tsx]: whitespace prior `</span>` ?

File: src/components/menu/menu.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    setToggleDisabled(true)
+    setCollapsed(!collapsed)
+
+    setTimeout(() => {
+      setToggleDisabled(false)
+    }, 300)
+  }
+
+  return (
+    <>
```
Comment (MateuszWasik) [inline src/components/menu/menu.tsx:59]: what does that do ?

**Review by MateuszWasik — APPROVED** [review]

File: src/components/menu/menu.tsx (hunk 4 above)
Comment (skrzepij) [reply src/components/menu/menu.tsx:59]: React component, when you define html template, requires single main element. 
Earlier almost always we use a `div` element to wrap multiple nested tags. But this generate lots of unnecessary nesting in rendered html file.

Thats why they implemented `<>` - which does not generate anything at all :D 

Lets say this is something like `<ng-container>` in angular xD

File: src/components/menu/menu.module.scss (hunk 2 above)
Comment (skrzepij) [reply src/components/menu/menu.module.scss]: this provides extra security. protects from CRSS SCSS Injectors

Comment (skrzepij) [reply src/components/menu/menu.module.scss]: Fixed mate bro thanks

File: src/components/menu/menu.tsx (hunk 4 above)
Comment (skrzepij) [reply src/components/menu/menu.tsx:59]: can we close this one please?

File: src/components/menu/menu.tsx (hunk 3 above)
Comment (skrzepij) [reply src/components/menu/menu.tsx]: fixed thanks

File: src/components/menu/menu.tsx (hunk 4 above)
Comment (MateuszWasik) [reply src/components/menu/menu.tsx:59]: Hard to say

**Pull request merged**